include/ola/client/ClientArgs.h

```diff
@@ -90,9 +90,9 @@ struct SendDMXArgs {
   /**
    * @brief Create a new SendDMXArgs object
    */
-  explicit SendDMXArgs(GeneralSetCallback *callback)
+  explicit SendDMXArgs(GeneralSetCallback *_callback)
```

Comment:
We normally name the field m_, would you mind doing that please and dropping the leading underscores from the constructor arguments.